src/main/java/uk/gov/hmcts/divorce/common/notification/FinalOrderRequestedNotification.java

```diff
@@ -109,4 +126,8 @@ private Map<String, String> solicitorsFinalOrderTemplateVars(final CaseData case
 
         return templateVars;
     }
+
+    private String getUniqueOverdueContent(String fullName, String delayReason) {
+        return fullName + DELAY_REASON_STATIC_CONTENT + delayReason;
```

Comment:
Rather than adding this method you could have done:

```
.map(reason -> DELAY_REASON.formatted(caseData.getApplicant1().getFullName(), reason))
```

Where DELAY_REASON is:

```
%s applied more than 12 months after ... following reason: \n %s"
```

Reply:
Done